docs/4.x/tests-js-tracker.md

```diff
@@ -1,7 +1,7 @@
 ---
 category: Develop
 previous: tests-ui
-next: tests-travis
+next: tests-github
```

Comment:
those need to be revert again as the Actions guide will be in a different category